mmv1/third_party/terraform/tests/resource_artifact_registry_repository_test.go.erb

```diff
@@ -39,6 +39,50 @@ func TestAccArtifactRegistryRepository_update(t *testing.T) {
 	})
 }
 
+func TestArtifactRegistryRepository_create_mvn_snapshot(t *testing.T) {
+	t.Parallel()
+
+	repositoryID := fmt.Sprintf("tf-test-%d", randInt(t))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
+		CheckDestroy: testAccCheckArtifactRegistryRepositoryDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccArtifactRegistryRepository_create(repositoryID, "SNAPSHOT"),
+			},
+			{
+				ResourceName:      "google_artifact_registry_repository.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestArtifactRegistryRepository_create_mvn_release(t *testing.T) {
```

Comment:
Needs `TestAcc` to get picked up by standard test runs.

```suggestion
func TestAccArtifactRegistryRepository_create_mvn_release(t *testing.T) {
```

Reply:
Done, thanks!